runtime/src/static_ids.rs

```diff
@@ -1,13 +1,12 @@
 use {
-    solana_inline_spl::{associated_token_account, token, token_2022},
     solana_pubkey::Pubkey,
+    spl_generic_token::{associated_token_account, token, token_2022},
 };
 
 lazy_static! {
     /// Vector of static token & mint IDs
     pub static ref STATIC_IDS: Vec<Pubkey> = vec![
         associated_token_account::id(),
-        associated_token_account::program_v1_1_0::id(),
         token::id(),
         token::native_mint::id(),
```

Comment:
note the only place this static is used is for snapshot reduction, and the old ata id does not exist on mainnet. we moved this id and the old token program id from `spl-generic-token`

Reply:
btw - loop is for snapshot minimization not just reducing snapshot size.

It's a feature in ledger-tool, you can basically trim a full-sized snapshot down into only the accounts you need to replay a specific slot range.